File: swagger-client/dart/lib/model/email_address_names.dart
part of swagger.api;

class EmailAddressNames {
  
  EmailAddressNames();

  @override
  String toString() {
    return 'EmailAddressNames[]';
  }

  EmailAddressNames.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
  }

  Map<String, dynamic> toJson() {
    return {
     };
  }

  static List<EmailAddressNames> listFromJson(List<dynamic> json) {
    return json == null ? new List<EmailAddressNames>() : json.map((value) => new EmailAddressNames.fromJson(value)).toList();
  }

  static Map<String, EmailAddressNames> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, EmailAddressNames>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new EmailAddressNames.fromJson(value));
    }
    return map;
  }
}

File: openapi-client/cpp-tiny/lib/Models/EmailAddressNames.cpp


#include "EmailAddressNames.h"

using namespace Tiny;

EmailAddressNames::EmailAddressNames()
{
}

EmailAddressNames::EmailAddressNames(std::string jsonString)
{
	this->fromJson(jsonString);
}

EmailAddressNames::~EmailAddressNames()
{

}

void
EmailAddressNames::fromJson(std::string jsonObj)
{
    bourne::json object = bourne::json::parse(jsonObj);


}

bourne::json
EmailAddressNames::toJson()
{
    bourne::json object = bourne::json::object();


    return object;

}



